src/vs/workbench/browser/parts/editor/editorTabsControl.ts

```diff
@@ -162,6 +164,8 @@ export abstract class EditorTabsControl extends Themable implements IEditorTabsC
 
 		this.renderDropdownAsChildElement = false;
 
```

Comment:
I wonder if we should use instant hovers for tabs

Reply:
Not for the meantime.